src/world/Objects/Units/Players/Player.h

```diff
@@ -1502,6 +1504,9 @@ class SERVER_DECL Player : public Unit
         void ApplyItemMods(Item* item, int16 slot, bool apply, bool justdrokedown = false) { _ApplyItemMods(item, slot, apply, justdrokedown); }
     protected:
         void _ApplyItemMods(Item* item, int16 slot, bool apply, bool justdrokedown = false, bool skip_stat_apply = false);
+#if VERSION_STRING > TBC
+        void calculateHeilroomBonus(ItemProperties const* proto, int16_t slot, bool apply);
```

Comment:
Move this function declaration to MIT license content. Since this is related to items, put it somewhere in between lines 797 and 813.